mmocr/datasets/preparers/gatherers/base.py

```diff
@@ -10,21 +10,30 @@ class BaseGatherer:
     directory and all the image files are in the same directory.
 
     Args:
+        img_dir(str): The directory of the images. It usually be set
```

Comment:
`It usually be` -> `It is usually`, same for `split` and `data_root` and `ann_dir`